==== triton/third_party/nvidia/BUILD#6 - triton/third_party/nvidia/BUILD ====
# action=edit type=text
--- triton/third_party/nvidia/BUILD	2024-06-17 05:34:30.000000000 -0700
+++ triton/third_party/nvidia/BUILD	2024-06-28 02:41:49.000000000 -0700
@@ -274,7 +274,6 @@
         ],
     }),
     includes = ["include"],
-    visibility = ["//:__subpackages__"],
     deps = [
         ":nvgpu_attr_inc_gen",
         ":nvgpu_dialect_inc_gen",
